tests/security/test_security_multistream.py

```diff
@@ -1,14 +1,13 @@
 import asyncio
 
-import multiaddr
-import pytest
-
 from libp2p import new_node
 from libp2p.peer.peerinfo import info_from_p2p_addr
 from libp2p.protocol_muxer.multiselect_client import MultiselectClientError
-from libp2p.security.insecure_security import InsecureTransport
-from libp2p.security.simple_security import SimpleSecurityTransport
-from tests.utils import cleanup, connect
+from libp2p.security.insecure.transport import InsecureSession, InsecureTransport
+from libp2p.security.simple.transport import SimpleSecurityTransport
+import multiaddr
+import pytest
```

Comment:
Is this sorting(`libp2p` above `multiaddr` and `pytest`) desired by us? If not, we probably need to adjust the configuration of isort a little bit.

Reply:
yeah i noticed this -- not sure what is going on here

Reply:
this seems to have magically fixed itself after i rebased master in #250.